x-pack/test/functional/apps/dashboard/feature_controls/dashboard_security.ts

```diff
@@ -84,7 +83,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
       it('shows dashboard navlink', async () => {
         const navLinks = await appsMenu.readLinks();
-        expect(navLinks.map((link) => link.text)).to.eql(['Dashboard', 'Stack Management']);
+        expect(navLinks.map((link) => link.text)).to.contain('Dashboard');
```

Comment:
This was failing because the [observability app is always showing now](https://github.com/elastic/kibana/issues/71164), which is a change in behavior.